docs/openapi/components/schemas/common/Organization.yml

```diff
@@ -115,20 +101,72 @@ properties:
     $linkedData:
       term: iataCarrierCode
       '@id': https://onerecord.iata.org/cargo/Company#airlineCode
+  organizationType:
+    title: Organization Type
+    description: Type of organization (e.g. cold storage, meat processing).
+    type: string
+    $linkedData:
+      term: organizationType
+      '@id': https://w3id.org/traceability#organizationType
+  place:
```

Comment:
@rhofvendahl, I love the ambition of normalizing our data model and applaud the effort in working through the cascading impacts! But replacing an attribute which is defined on schema.org (`Organization.address`) with one which isn't `Organization.place` doesn't seem right.
The attribute should be `Organization.location`, ref: https://schema.org/Organization